winforms/src/toga_winforms/widgets/canvas.py

```diff
@@ -300,16 +300,24 @@ def reset_transform(self, draw_context, **kwargs):
         self.scale(self.dpi_scale, self.dpi_scale, draw_context)
 
     # Text
-    def write_text(self, text, x, y, font, baseline, draw_context, **kwargs):
+    def _line_height(self, font, line_height):
+        if line_height is None:
+            return font.metric("LineSpacing")
+        else:
+            return font.metric("EmHeight") * line_height
```

Comment:
Same issue here - but I'm more certain Em height isn't the same as font size.

Reply:
Huh... I thought em-width was variable with the font, but evidently not. Regardless - as you say, we have the actual font size, so we can use that directly.